### DISCUSSION TITLE
temperature device for monitoring time-temperature ISO 834

### DISCUSSION THREAD

**Discussion by Prometeo-ianic** [firemodels/fds#12883] [General]
Hello everyone, I have these questions: 
1.	Are the temperature-time curves like ISO 834 during furnace tests measured as gas temperature or surface temperature? Is the actual behavior of these curves monitored with flat thermometers (similar to ISO 834) or gas temperature probes? 
2. How can we ensure that the temperature (gas or surface) is equal throughout the furnace? Is there a conventional method? 
 
I am asking these questions for two reasons: 
1.	I would like to create a sort of virtual furnace on FDS to better understand the functioning of the tests. 
2. .	I have been wondering for some time what parameter would be most consistent to provide for fire resistance purposes when using finite element models. In particular, I have been using adiabatic temperature device in FDS for some time, following the indications provided here: 
Wickström, Ulf & Duthinh, D. & McGrattan, K.. (2007). Adiabatic surface temperature for calculating heat transfer to fire exposed structures. Interflam 2007. 2. 943-953. 
Even from a regulatory standpoint, when referring to temperature-time curves, gas temperature is understood rather than adiabatic temperature or net heat flux. To delve deeper into this matter, I have looked into the physics behind furnace tests. It seems to me that curves like ISO 834 result from measurements with pyrometers (in early 1900s tests) and plate thermometers (recently), which I believe are more closely related to adiabatic surface temperature (according to the previously cited publication) and less consistent with gas temperature. However, I would appreciate a comparison in this regard. 
Thank you, Regards

**Comment by drjfloyd** [comment]
The temperature-time curve is a gas temperature curve. How to measure temperature is detailed in the specific standard being used.  

In the ASTM E119 standard, mulitple inconnel sheathed, type-K thermocouples are used. These are placed around the sample with the average value used to control the furnace with the allowable error being a function of the test duration (longer tests have lower allowable error recognizing that the early rapid ramp may be challenging to achieve).

The large furnaces used for testing wall and floor assemblies use mulitple, pre-mixed burners. The flow from the burners and the use of multiple burners helps to mix the air volume. The walls also help with uniformity as walls near higher temperature regions will radiate heat to walls near lower temperature regions heating the lower temperature walls and convecting heat back into the gas.

I did a presentation for the 2018 FEMTC on the control functions in FDS using a simplified furnace geometry as one example: https://files.thunderheadeng.com/femtc/2018_d3-01-floyd-paper.pdf.

**Reply by Prometeo-ianic** [reply]
Thank you very much for the answer and for your reference! 
Just an other question: which is the purpose of the plate thermometer if the temperature is controlled by the inconnel sheathed type-K thermocouples?

**Reply by drjfloyd** [reply]
My statement began with "In the ASTM E119 standard". The standard applicable for your specific needs may differ. You need to consult the text of the standard that applies to your specific situation. If the applicable standard calls for the use of Type-K thermocouples, then you need to use type-K thermocouples and not plate thermocouples.  If the applicable standard calls for the use of plate thermometers, then you need to use plate thermometers constructed according to the requirements of the standard and not type-K thermocouples.

**Reply by Prometeo-ianic** [reply]
Can the inconnel sheathed, type-K thermocouples be modelled as gas temperature devices? Is the furnace a case where adiabatic gas temperature, gas temperature, tend to the same value (after some minutes)? Looking at this paper I had these impressions
Sultan, M.A. Performance of Different Temperature Sensors in Standard Fire Resistance Test Furnaces. Fire Technol 46, 853–881 (2010).

**Reply by mcgratta** [reply]
There is an output quantity called `THERMOCOUPLE`. Search the User's Guide for details.

**Reply by Prometeo-ianic** [reply]
Just a curiosity, since you have a large experience on validation, which thermocouple can produce similar value to gas temperature device? I had the impression that avoiding the radiating component of heat could be very challenging, but I see that in the validation guide there are a lot of validation cases that involve gas temperature.

**Reply by drjfloyd** [reply]
Some reasons why TEMPERATURE may have been used instead of THERMOCOUPLE:
-If you aspirate a thermocouple you can get something close to the gas temperature.
-Some test data has radiation corrected temperatures.
-If temperatures are low and the radiation from the fire is not large at the TC location, then the radiation error will be small.
-May also be cases were we should have used THERMOCOUPLE but used TEMPERATURE
